app/v1/models/hateoas/RelType.scala

```diff
@@ -18,10 +18,12 @@ package v1.models.hateoas
 
 object RelType {
   val SELF = "self"
-  val AMEND_RELIEF_INVESTMENTS = "amend-relief-investments"
-  val DELETE_RELIEF_INVESTMENTS = "delete-relief-investments"
+  val AMEND_RELIEF_INVESTMENTS = "amend-reliefs-investments"
+  val DELETE_RELIEF_INVESTMENTS = "delete-reliefs-investments"
```

Comment:
Should have the rel type examples in RAML also changed to match up with this change.